.github/workflows/build-with-bal-test-graalvm-template.yml

```diff
@@ -38,19 +38,11 @@ jobs:
         if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'Skip GraalVM Check')
 
         steps:
-            - name: Set up GraalVM
-              uses: graalvm/setup-graalvm@v1
+            - name: Set up JDK 17
+              uses: actions/setup-java@v3
               with:
-                  java-version: '17.0.7'
-                  distribution: 'graalvm'
-                  components: 'native-image'
-                  github-token: ${{ secrets.GITHUB_TOKEN }}
-
-            - name: Check GraalVM installation
-              run: |
-                  echo "GRAALVM_HOME: ${{ env.GRAALVM_HOME }}"
-                  echo "JAVA_HOME: ${{ env.JAVA_HOME }}"
-                  native-image --version
+                  distribution: 'temurin'
+                  java-version: '17'
```

Comment:
```suggestion
                  java-version: '17.0.7'
```

Reply:
Lets try with the latest version, if there is an issue will change this version